cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -675,6 +679,95 @@ def test_can_get_correct_usage_info_for_content_groups(self):
 
         self.assertEqual(actual, expected)
 
+    @ddt.data(ModuleStoreEnum.Type.split)
```

Comment:
I think I'd test this in both split and mongo, and define expected behavior for each, sort of like what you did for SUST-20